### PR TITLE
Added support for dlg.stats_active module

### PR DESCRIPTION
**Pull request by faizann**
Kindly merge and release a new version.

### CODE REVIEW

**Comment by florentchauveau** [comment]
Hello,

I have merged another pull request, could you rebase and resubmit?

I will also comment some changes.

Thank you :)

File: README.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ Flags:
                              URI on which to scrape kamailio. E.g.
                              "unix:/var/run/kamailio/kamailio_ctl" or
                              "tcp://localhost:2049"
-  -m, --kamailio.methods="tm.stats,sl.stats,core.shmmem,core.uptime"
+  -m, --kamailio.methods="tm.stats,sl.stats,core.shmmem,core.uptime,dlg.stats_active"
```
Comment (florentchauveau) [inline README.md]: This is the "default" metrics, and as "dlg" requires the dialog module, I think we should not enable this metric by default. Plus, you did not enabled it as a default metric in `main.go` (which is good).

File: README.md [hunk 2]
```diff
@@ -34,7 +34,7 @@ Flags:
...
                              Comma-separated list of methods to call. E.g.
                              "tm.stats,sl.stats". Implemented:
                              tm.stats,sl.stats,core.shmmem,core.uptime,dispatcher.list
```
Comment (florentchauveau) [inline README.md]: Please add `dlg.stats_active` here instead :)

File: README.md [hunk 3]
```diff
@@ -59,6 +59,7 @@ By default, the exporter will try to fetch values from the following commands:
 - `sl.stats` (requires the [SL](http://kamailio.org/docs/modules/stable/modules/sl.html) module)
 - `core.shmmem`
 - `core.uptime`
+- `dlg.stats_active` (requires the [DLG](http://kamailio.org/docs/modules/stable/modules/dialog.html) module)
```
Comment (florentchauveau) [inline README.md]: Default metrics here, so it should not be listed here (see below)

File: README.md [hunk 4]
```diff
@@ -59,6 +59,7 @@ By default, the exporter will try to fetch values from the following commands:
...
 
 If you are using the [DISPATCHER](http://kamailio.org/docs/modules/stable/modules/dispatcher.html) module, you can enable `dispatcher.list` as well:
```
Comment (florentchauveau) [inline README.md]: Maybe you can add a section like this : "If you are using the dialog module..."? What do you think?

Comment (faizann) [reply README.md]: Yeah, I reformatted it a bit and also added missing info about TLS module. 
I guess a simple table for module -> param would be enough at some point.

**Comment by faizann** [comment]
Resubmitting the MR with conflicts resolved. 
Thanks

**Comment by florentchauveau** [comment]
Awesome! Thank you

**Pull request merged**